Utils/TimeConstants.swift

```diff
@@ -18,6 +18,45 @@ extension NSDate {
     public class func now() -> Timestamp {
         return UInt64(1000 * NSDate().timeIntervalSince1970)
     }
+    
+    public func toRelativeTimeString() -> String {
+        let now = NSDate()
+        let units = NSCalendarUnit.CalendarUnitSecond | NSCalendarUnit.CalendarUnitMinute | NSCalendarUnit.CalendarUnitDay | NSCalendarUnit.CalendarUnitWeekOfYear | NSCalendarUnit.CalendarUnitMonth | NSCalendarUnit.CalendarUnitYear | NSCalendarUnit.CalendarUnitHour
+        let components = NSCalendar.currentCalendar().components(units, fromDate: self, toDate: now, options: NSCalendarOptions.allZeros)
+        let formatter = NSDateFormatter()
+        
+        if components.year > 0 {
+            return String(format: NSLocalizedString("%@", comment: "Relative date for date older than an year."), NSDateFormatter.localizedStringFromDate(self, dateStyle: NSDateFormatterStyle.ShortStyle, timeStyle: NSDateFormatterStyle.ShortStyle))
```

Comment:
This shouldn't be a localized string; it's an absolute date, not a relative date, so NSDateFormatter will take care of this for us. Just use the last half of this line on its own.